frontend/scripts/react-components/explore/top.component.jsx

```diff
@@ -4,67 +4,39 @@ import PropTypes from 'prop-types';
 import Link from 'redux-first-router-link';
 import Placehold from 'react-placehodl';
 import cx from 'classnames';
-import formatValue from 'utils/formatValue';
 import 'styles/components/shared/top.scss';
 
 class Top extends Component {
-  static getValue(value, format) {
-    const formatFn = !format ? x => formatValue(x * 100, 'percentage') : formatValue;
-    return Array.isArray(value) ? formatFn(value[0], format) : formatFn(value, format);
```

Comment:
@sorodrigo was that `isArray` needed here?

Reply:
It's something that was needed before in the profile pages. Thanks